share/models/meta.py

```diff
@@ -26,6 +26,7 @@ class Award(ShareObject):
     award = ShareURLField(blank=True)
     description = models.TextField(blank=True)
     url = ShareURLField(blank=True)
+    funder = ShareManyToManyField('Funder', through='ThroughAwardFunders')
```

Comment:
I can't really see any reason not to. I was just doing what the group suggested. Maybe an award could be funded by an Organization or Publication. I don't know. They just mentioned Funder.